homotopy-graphics/src/style.rs

```diff
@@ -49,6 +49,8 @@ impl FromStr for Color {
 }
 
 // Reverse of `FromStr`
+// NOTE: If we decide to change how we `fmt::Display`, it could break some styles as it is used
```

Comment:
Maybe make a specific method for manim export colour formatting to avoid this?

Reply:
Yeah I think that's probably safer.
Orignally, it was implemented like that but we found that we ended up with a bunch of repeated definitions for the `hex` functions for each renderer.
In a previous comment @calintat requested this be reduced into one canonical `impl` method.

I'm honestly very happy to create those functions again for clarity. I think the easiest fix would probably be to create an actual `impl` method `hex` on top of Color and that way there is no dispute over which format should be returned.

Reply:
This is what I have implemented for now. As far as I can tell, this is otherwise ready to merge so I'll wait for confirmation or further comment before squash and merge.

Reply:
This is better, thanks!